dashboards/state-sync-v2.json

```diff
@@ -0,0 +1,3546 @@
+{
```

Comment:
@rustielin, silly question: What's the reason this dashboard is shown as new and not moved from terraform/helm/validator/files/dashboards/? We should already have one in there today, right?

Reply:
good question. i think it might have to do with how I rebased a few times to keep up with the dashboard changes in main. manually comparing and diff only shows the newline change

Reply:
```
wget https://raw.githubusercontent.com/aptos-labs/aptos-core/6fa8a8b45f173f8cb9ab1207a19ab3876264fb73/dashboards/state-sync-v2.json
wget https://raw.githubusercontent.com/aptos-labs/aptos-core/main/terraform/helm/validator/files/dashboards/state-sync-v2.json
diff state-sync-v2.json state-sync-v2.json.1
```

Reply:
Cool! Thanks for checking 😄